node/hyperbahn/service_proxy.js

```diff
@@ -124,6 +124,9 @@ ServiceDispatchHandler.prototype.rateLimit =
 function rateLimit(req, buildRes) {
     var self = this;
 
+    // stats all traffic using rate limiter's counter
+    self.rateLimiter.incrementServiceCounter(req.headers.cn + '~~' + req.serviceName);
```

Comment:
I think it's bad to make this a service counter. 

I think its better to have a new bucket in the rate limiter thats not "ServiceCounter" but is instead "EdgeCounter"